macros/src/main/scala/me/rexim/morganey/meta/package.scala

```diff
@@ -0,0 +1,19 @@
+package me.rexim.morganey
+
+import scala.language.experimental.macros
+
+package object meta extends AnyRef
+  with DefaultLiftableInstances
+  with DefaultUnliftableInstances {
+
+  private[meta] trait Quasiquotation {
+    def apply[T](args: T*): Any = macro QuotationMacro.quote
+    def unapply(arg: Any): Any = macro QuotationMacro.unquote
+  }
+
+  implicit class Quasiquote(ctx: StringContext) {
+    object m extends Quasiquotation
+    object lc extends Quasiquotation
```

Comment:
What do we need the `lc` interpolator for? It looks like it does the same thing as `m`, no?